requirements/windows.txt

```diff
@@ -3,3 +3,34 @@
 # If they are windows specific, place "; sys_platform == 'win32'" in front of the requirement.
 
 -r zeromq.txt
+# <<<<<<< HEAD
```

Comment:
got some need human to resolve conflicts here

Reply:
Left this here while I validated the dependencies after migrating them to `base.txt`. I will clean it up before merging anything.

Reply:
Wait I thought it was like green M&Ms on the rider, checking if we review :rofl: